paper/paper.md

```diff
@@ -0,0 +1,171 @@
+
+• Simulation class adds a layer on top of the HOOMD-blue simulation object, which
+adds additional methods and features that simplifies the process of starting and
+resuming a HOOMD-blue simulation.
+
+Additionally, `JankFlow` offers a library pre-defined subclasses of the above base classes
+including common polymers, forcefields and bulk system initialization algorithms.
+
+# Recipes
+`JankFlow` offers the following two ready-to-go recipes to illustrate how the design creates
+potential for expanding the library of open-source and version-controlled workflows.
+
+• Welding: What does this recipe do. Simulation to create slabs, building up an
+interface from slabs, simulation to preform welding.
+• Tensile Testing
+```python
+from jankflow.library import PolyEthylene, OPLS_AA, Tensile
+from jankflow.base import Pack
+from jankflow.modules.welding import SlabSimulation, Interface, WeldSimulation
+
+# initialize a polymer system using poly(ethylene) and OPLS-AA forcefield
+molecule = PolyEthylene(num_mols=30, lengths=12)
+system = Pack(molecules=molecule, density=1.1)
+system.apply_forcefield(r_cut=2.5, force_field=OPLS_AA(), auto_scale=True,
+                        remove_charges=True, remove_hydrogens=True)
+# saving forces for later use
+hoomd_forces = system.hoomd_forcefield
+# run a slab simulation
+sim = SlabSimulation.from_system(system=system, interface_axis=(1, 0, 0),
+                                 gsd_file_name="slab.gsd")
+# shrink the box to reach the desired density
+sim.run_update_volume(final_density=1.2, n_steps=5e4, kT=5.0,
+                      period=100, tau_kt=0.001)
+# run NVT ensemble
+sim.run_NVT(kT=5.0, n_steps=4e4, tau_kt=0.001)
+sim.operations.writers[0].flush()
+# create an interface from the slab
+interface = Interface(gsd_file="slab.gsd", interface_axis=(1, 0, 0), gap=0.05)
+# run the welding simulation
+weld_sim = WeldSimulation(initial_state=interface.hoomd_snapshot,
+                          forcefield=hoomd_forces, interface_axis=(1, 0, 0),
+                          gsd_file_name="weld.gsd", log_file_name="weld_log.txt",
+                          log_write_freq=500, dt=0.0003)
+weld_sim.run_NVT(kT=10.0, n_steps=7e4, tau_kt=0.001)
+cooling_ramp = weld_sim.temperature_ramp(n_steps=2e4, kT_start=10.0, kT_final=2.0)
+weld_sim.run_NVT(kT=cooling_ramp, n_steps=2e4, tau_kt=0.001)
+weld_sim.save_restart_gsd("weld_restart.gsd")
+# Running a tensile test simulation
+tensile_sim = Tensile(initial_state="weld_restart.gsd", forcefield=hoomd_forces,
+                      tensile_axis=(1,0,0), gsd_file_name="tensile.gsd",
+                      gsd_write_freq=1000, log_file_name="tensile_log.txt",
+                      log_write_freq=500, fix_ratio=0.30)
+tensile_sim.run_tensile(n_steps=1e5, strain=0.70, period=500, kT=2.0, tau_kt=0.001)
+# for more details on the tensile test results, please see the tutorials.
+```
+
+# Availability
+`JankFlow` is freely available under the GNU General Public License (version 3)
```

Comment:
@erjank We are using [GPL3 ](https://choosealicense.com/licenses/gpl-3.0/)license right now but I see many JOSS papers use [MIT](https://choosealicense.com/licenses/mit/). 
GLP3 sounds a little too loose IMO.

Reply:
I think technically GPL has more restrictions than MIT? I believe GPL says that if something uses this code, it has to be free and open source as well.  I don't think MIT has that same restriction (e.g. someone could make a GUI for jankflow and charge people to use it) I don't have a strong preference on the license we use as long as there are not conflicts with other packages (are we using any packages that are GPL?)